File: Tugas_2/backtracking/main.go
package main

import (
	"backtracking/mrv"
	"fmt"
)

func main() {
	var whichAlgorithm string
	fmt.Println("1. MRV")
	fmt.Scanln(&whichAlgorithm)

	switch whichAlgorithm {
	case "1":
		mrv.MRV()
	}

}

File: Tugas_2/backtracking/main.py
from mrv.mrv import MRV
from degreeHeuristic.degreeHeuristic import Degree_Heuristic

def main():
    whichAlgorithm = input("1. MRV\n2. Degree Heuristic\npilih salah satu: ")
    if whichAlgorithm == "1":
        MRV()
    elif whichAlgorithm == "2":
        Degree_Heuristic()

if __name__ == "__main__":
    main()
